src/lib/autoUpdater.js

```diff
@@ -0,0 +1,31 @@
+
+const { app, autoUpdater } = require('electron');
+
+const SERVER = 'https://update.electronjs.org';
+
+class AutoUpdater {
+  constructor(options = {}) {
+    const feed = `${SERVER}/headsetapp/headset-electron/${process.platform}-${process.arch}/${app.getVersion()}`;
+
+    autoUpdater.setFeedURL(feed);
+
+
+    autoUpdater.on('update-downloaded', () => {
+      options.onUpdateDownloaded();
+    });
+
+    autoUpdater.checkForUpdates();
```

Comment:
We can't call this method is the app is not installed on Windows, so we need to check if the app is installed first before running this. That's why it's not passing tests on Appveyor.